tests/ocf-url_link_leaking_relative/EPUB/content_001.xhtml

```diff
@@ -0,0 +1,24 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+  <head>
+    <title>Potentially leaking relative link</title>
+    <style>
+      figure {
+        text-align: center
+      }
+      img {
+        width: 60%;
+      }
+      figcaption {
+        font-size: smaller;
+        font-style: italic;
+      }
+    </style>
```

Comment:
This CSS is fine but seems unnecessary; won't affect the test, so I would omit.

Reply:
ok